### PR TITLE
#677 DOM manipulation research questions added

### PR DESCRIPTION
**Pull request by rachaelcodes**
I've added quite a lot of questions, but hopefully some of this was covered in the pre-course materials, so groups could work through the first few quite quickly. I had a look at FAC11's project READMEs, and these were the DOM areas mentioned.

relates #677 and #629

### CODE REVIEW

File: coursebook/week-2/research-afternoon.md [hunk 1]
```diff
@@ -11,10 +11,12 @@
...
-### Software development methodologies
-- What is the traditional software development life-cycle?
-- How does agile differ from traditional waterfall methodologies?
-- How does the general software development life-cycle differ from the TDD cycle?
+### DOM manipulation
+- How can you use JavaScript to create an HTML element and then add it to your webpage?
+- How can you use JavaScript to replace an existing element on a webpage with a new one?
+- How would you add a `<li>` element to the start of a `<ul>`?
+- What is a JavaScript Event? What does event.preventDefault() and why might you use it?
+- What are the security concerns around `Element.innerHTML` and what could you use instead?
```
Comment (eliascodes) [inline coursebook/week-2/research-afternoon.md]: Worth also having something about distinction between `Array` and `NodeList`?

Comment (rachaelcodes) [reply coursebook/week-2/research-afternoon.md]: good call - I've added it in the new version

File: coursebook/week-2/research-afternoon.md [hunk 2]
```diff
@@ -11,10 +11,12 @@
...
 - Give an example of how a spike could have helped with last week's projects.
 
-### Software development methodologies
-- What is the traditional software development life-cycle?
-- How does agile differ from traditional waterfall methodologies?
-- How does the general software development life-cycle differ from the TDD cycle?
+### DOM manipulation
+- How can you use JavaScript to create an HTML element and then add it to your webpage? How would you replace an existing element with it?
+- How would you add a `<li>` element to the start of a `<ul>`?
+- What is a JavaScript Event? What does event.preventDefault() and why might you use it?
```
Comment (ZooeyMiller) [inline coursebook/week-2/research-afternoon.md]: "What does event.preventDefault() and why might you use it?" missing the word "do".

**Comment by rachaelcodes** [comment]
Thanks for the spot @ZooeyMiller - it should now be updated.

**Pull request merged**